### PR TITLE
Added front deployer

### PR DESCRIPTION
**Pull request by thullSL**

### CODE REVIEW

File: bucketChucker.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	.then(uploadFiles);
+
+
+function readBuild(dir) {
+	console.log("Reading Directory " + dir);
+	return lib.promiseMaker(
+		fs, 
+		fs.readdir,
+		dir,
+		function(data) {return data},
```
Comment (csscottlogic) [inline bucketChucker.js]: This wouldn't lint unless it was ```return data;```

File: bucketChucker.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		function(data) {return "Read " + data.length + " files from build directory: ", data}
```
Comment (csscottlogic) [inline bucketChucker.js]: Does that comma operator throw away the "Read ... directory: " bit?

File: bucketChucker.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+var lib = require('./lib');
+AWS.config.update({region: 'eu-west-1'});
+var s3 = new AWS.S3();
+
+var args = process.argv.slice(2);
+var inputDir = args[0]
+var inputBucket = args[1]
+
+readBuild(inputDir)
+	.then(uploadFiles);
```
Comment (csscottlogic) [inline bucketChucker.js]: This looks like it's indented by 7 spaces?

File: bucketChucker.js (hunk 2 above)
Comment (thullSL) [inline bucketChucker.js]: It did, have fixed this.

File: bucketChucker.js (hunk 1 above)
Comment (thullSL) [inline bucketChucker.js]: Have put ; in in all the places like this.

**Comment by jebbinSCL** [comment]
:shipit:

**Comment by lclarksl** [comment]
:shipit:

**Pull request merged**